spack/packages/dla-future/package.py

```diff
@@ -29,24 +29,25 @@ class DlaFuture(CMakePackage):
 
     def cmake_args(self):
        spec = self.spec
-       
+
        if (spec.satisfies('^intel-mkl')):
            args = ['-DDLAF_WITH_MKL=ON']
        else:
            args = ['-DDLAF_WITH_MKL=OFF']
+           lapack_name = spec['lapack'].libs.ld_flags.split()[1]
+           blas_name = spec['blas'].libs.ld_flags.split()[1]
+           lapack_libs = spec['lapack'].prefix.lib
+           blas_libs = spec['blas'].prefix.lib
            args.append('-DLAPACK_TYPE=Custom')
-           args.append('-DLAPACK_LIBRARY=-L{0} -lblas -llapack'.format(spec['lapack'].prefix.lib))
+           args.append('-DLAPACK_LIBRARY=-L{0} {1} -L{2} {3}'.format(lapack_libs, lapack_name, blas_libs, blas_name))
```

Comment:
Wouldn't be possible to have something like this?

```
-DLAPACK_LIBRARY='{} {}'".format(spec['lapack'].libs.ld_flags, spec['blas'].libs.ld_flags)
```

Reply:
sure